content/projects/kotlin/project-5/live-data-transformations/_index.md

```diff
@@ -0,0 +1,178 @@
+---
+available_flavours:
+- kotlin
+content_type: project
+prerequisites:
+  hard:
+  - projects/kotlin/project-5/data-binding-viewmodel-livedata
+  soft: []
+ready: true
+submission_type: repo
+title: LiveData transformations.
+---
+
+### App overview
+
+In the this project, you develop the GuessTheWord app, beginning with the previous project code. GuessTheWord is a two-player charades-style game, where the players collaborate to achieve the highest score possible.
+
+The first player looks at the words in the app and acts each one out in turn, making sure not to show the word to the second player. The second player tries to guess the word.
+
+To play the game, the first player opens the app on the device and sees a word, for example "guitar," as shown in the screenshot below.
+
+![](a.png)
+
+The first player acts out the word, being careful not to actually say the word itself.
+
+- When the second player guesses the word correctly, the first player presses the Got It button, which increases the count by one and shows the next word.
+- If the second player can't guess the word, the first player presses the Skip button, which decreases the count by one and skips to the next word.
+- To end the game, press the End Game button. (This functionality isn't in the starter code for the first codelab in the series.)
+
+In this project, you improve the GuessTheWord app by adding a one-minute countdown timer that appears above the score. The timer ends the game when the countdown reaches 0.
+
+You also use a transformation to format the elapsed time LiveData object into a timer string LiveData object. The transformed LiveData is the data binding source for the timer's text view.
+
+### Task: Get started
+
+In this task, you locate and run your starter code for this codelab. You can use the GuessTheWord app that you built in previous codelab as your starter code, or you can download a starter app.
```

Comment:
should this not be a `continue_repo` project?

Reply:
I fell it's better to have an instruction that is completely new to each project and a new repo so that they don't have an excuse to not start a new project. That is unfortunately the type of people I have to deal with.